cleanlab/multiannotator.py

```diff
@@ -92,10 +99,16 @@ def get_label_quality_multiannotator(
 
         * ``crowdlab``: an emsemble method that weighs both the annotators' labels as well as the model's prediction.
         * ``agreement``: the fraction of annotators that agree with the consensus label.
+    temp_scale : bool, default = False
+        Boolean value that specifies if the pred_probs will be temperature scaled to better match the annotators' empirical label distribution.
+        This is especially relevant in active learning to prevent overconfident models.
     return_detailed_quality: bool, default = True
         Boolean to specify if `detailed_label_quality` is returned.
     return_annotator_stats : bool, default = True
         Boolean to specify if `annotator_stats` is returned.
+    return_weights : bool, default = False
+        Boolean to specify if `model_weight` and `annotator_weight` is returned.
+        Model and annotator weights are applicable for ``quality_method == crowdlab``, will return ``None`` for any other quality methods.
```

Comment:
what about just throwing ValueError if return_weights is True but quality_method is not crowdlab?

Reply:
Imo there is no need to throw an error, since all the other outputs will still be computed correctly, and I think it will be nice to return those to users regardless. Also, `return_weights` is by default `False`, and `quality_method` is by default crowdlab so I don't think that scenario will happen often? 
Instead of an error perhaps we can throw a warning?

Reply:
> return_weights is by default False, and quality_method is by default crowdlab so I don't think that scenario will happen often

That was the reasoning behind my point. Why is somebody overriding return_weights when they are using a different quality method?  If they have done this, then IMO they are misunderstanding something and should get an error message.

Is there a downside to giving them the error message? Won't they just get the error message immediately and can just call the function again with return_weights = False this time?

Reply:
Currently the `if return_weights == True` check is done at the very end (after everything is computed and right before returning the dictionary with results), so in the current setup everything will already be computed.
It is possible to throw an error immediately, I just need a add a check at the beginning of the function before doing the computes.

Reply:
Ya I would just add that at the beginning. Want to be suspicious of a user overriding defaults in a nonsensical manner